tests/test_configure_feedstock.py

```diff
@@ -1,4 +1,6 @@
 import os
+
+from jsonschema import ValidationError
```

Comment:
With a shiny new JSON schema in hand, one can go _ape_ with stuff like https://github.com/python-jsonschema/hypothesis-jsonschema